api/src/main/kotlin/com/google/devtools/ksp/processing/Resolver.kt

```diff
@@ -141,4 +141,33 @@ interface Resolver {
         function: KSFunctionDeclaration,
         containing: KSType
     ): KSFunction
+
+    /**
+     * Returns the jvm name of the given function.
```

Comment:
Can you also mention that this ties to Kotlin compiler's implementation and is not guaranteed to be cross-version compatible? Namely, not guaranteed to work on libraries compiled by other version of compilers.

Reply:
updated. lmk.